pkg/registry/shadow/template/rest.go

```diff
@@ -160,6 +255,47 @@ func (r *REST) Get(ctx context.Context, name string, options *metav1.GetOptions)
 	return transformManifest(manifest)
 }
 
+// GetFromCCls retrieves the item from child cls.
+func (r *REST) GetFromCCls(ctx context.Context, name string, options *metav1.GetOptions) (runtime.Object, error) {
+	mcls, err := r.getMCLs()
+	if err != nil {
+		return nil, apierrors.NewServiceUnavailable(err.Error())
+	}
+	resource, _ := r.getResourceName()
+	return r.getResourceFromCCls(ctx, strings.ToLower(resource), name, options, mcls)
+}
+
+// getResourceFromCCls get resource From child cluster
+func (r *REST) getResourceFromCCls(ctx context.Context, resource string, name string, options *metav1.GetOptions,
+	mcls []*v1beta1.ManagedCluster) (runtime.Object, error) {
+	var unstructObj *unstructured.Unstructured
+	var err error
+	for _, cls := range mcls {
+		clusterID := string(cls.Spec.ClusterID)
+		clientset, ok := r.getCClsCliFromCache(clusterID)
+		if !ok {
+			clientset, err = r.newCClsCLi(clusterID, cls)
+			if err != nil {
+				return nil, apierrors.NewServiceUnavailable(err.Error())
+			}
+		}
+		namespace := request.NamespaceValue(ctx)
+		gvr := schema.GroupVersionResource{Version: "v1", Resource: resource}
+		unstructObj, err = clientset.
+			Resource(gvr).
+			Namespace(namespace).
+			Get(ctx, name, *options)
+		if err != nil {
+			klog.Errorf("get %s %s failed from %s: %v", resource, name, cls.Spec.ClusterID, err)
+			continue
+		} else {
+			err = nil
+			break
+		}
+	}
+	return unstructObj, err
```

Comment:
This seems to be wrong. I do NOT see any objects concentration here.

Here it just returns results from the last cluster.